.github/workflows/local-deployment.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       shell: bash
       run: |
         set -euxo pipefail
+        juju add-model jenkins
```

Comment:
Can we have a better name for this model?

Reply:
The only thing we do with this model is deploy Jenkins. What would you suggest to improve it?